src/components/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+type Props = {
+  value: number;
+  setValue: React.Dispatch<React.SetStateAction<number>>;
+  setPage: React.Dispatch<React.SetStateAction<number>>;
+};
+
+export const DropDown: React.FC<Props> = ({ value, setValue, setPage }) => {
+  const handleSelectChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    event.preventDefault();
+    setValue(Number(event.target.value));
+    setPage(1);
+  };
+
+  return (
+    <div className="form-group row">
+      <div className="col-3 col-sm-2 col-xl-1">
+        <select
+          value={value}
```

Comment:
The `value` prop of the select element should be controlled by a state or a prop to ensure that it reflects the current selection. Make sure that the `value` prop passed to `DropDown` is managed appropriately in the parent component.